robot/web/web_c.go

```diff
@@ -334,7 +334,7 @@ func (svc *webService) closeStreamServer() {
 	}
 }
 
-func (svc *webService) initStreamServer(ctx context.Context, options *weboptions.Options) error {
+func (svc *webService) initStreamServer(ctx context.Context, options *weboptions.Options, logger logging.Logger) error {
```

Comment:
is this addition needed?

Reply:
Nope! Good eye. I think these CGO-interacting files do not get linted for some reason.